kernel/src/main/scala/org/apache/toree/kernel/api/Kernel.scala

```diff
@@ -396,7 +372,29 @@ class Kernel (
     interpreterManager.interpreters.get(name)
   }
 
-  override def sparkSession: SparkSession = SparkSession.builder.getOrCreate
+  private lazy val defaultSparkConf: SparkConf = createSparkConf(
+    new SparkConf().setAppName(SparkKernelInfo.banner))
```

Comment:
I don't think we want to set the appName here since it conflicts with this recent PR: https://github.com/apache/incubator-toree/pull/107.